### ISSUE TITLE
Calling startActivity() from outside of an Activity

### ISSUE THREAD

**Issue by YUSMLE** [FarshadTahmasbi/EzPermission#1]
When I'm trying to grant permission(s), a runtime exception occurred that says calling `startActivity()` from outside of an Activity  context requires the `FLAG_ACTIVITY_NEW_TASK` flag. 

minimumSdkVersion = 23
compileSdkVersion = 28 
targetSdkVersion = 28

Snipped code to request permission(s): 

```
EzPermission.with(applicationContext)
            .permissions(
                Manifest.permission.WRITE_EXTERNAL_STORAGE,
                Manifest.permission.ACCESS_FINE_LOCATION
            )
            .request { granted, denied, permanentlyDenied ->
                granted.forEach { permission -> Log.w("Granted Permission", permission) }
                granted.forEach { permission -> Log.w("Denied Permission", permission) }
                granted.forEach { permission -> Log.w("Permanently Denied", permission) }
            }
```

**Comment by FarshadTahmasbi** [comment]
Yep, unfortunately I missed the `FLAG_ACTIVITY_NEW_TASK` flag for the intent, I'll fix and release an update ASAP.  Thanks for the comment.

**Comment by FarshadTahmasbi** [comment]
@YUSMLE By the way as a temporary solution try to pass an activity context instead of application context, It should work for now.

**Comment by FarshadTahmasbi** [comment]
Bug fixed on release 0.1.4, closing issue...
